pkg/collect/mysql.go

```diff
@@ -27,6 +27,38 @@ func Mysql(c *Collector, databaseCollector *troubleshootv1beta2.Database) (Colle
 			databaseConnection.IsConnected = true
 			databaseConnection.Version = version
 		}
+
+		rows, err := db.Query("SHOW VARIABLES")
+		if err != nil {
+			databaseConnection.Error = err.Error()
+		}
+		columns, err := rows.Columns()
+		if err != nil {
+			databaseConnection.Error = err.Error()
+		}
+		values := make([]sql.RawBytes, len(columns))
```

Comment:
I feel like we could clean this up.  I think `.Scan` can probably take a `map[string]interface{}`.

I'll do some tests tonight and report back.